v1/private/application/__init__.py

```diff
@@ -83,6 +94,8 @@ async def cabinet(user=Depends(auth_required)):
 
 @application_router.get("/cabinet/{reference}/status/")
 async def cabinet_status(reference: str, user=Depends(auth_required)):
+    # TODO: Use reference in pair with user_id to get the application status
+    # Because in future multiple applications will be allowed
     _user = await UserDoc.get(user.get("sub"))
     _ap = await ApplicationDoc.find_one({"user_id": _user.id})
```

Comment:
Application Reference should be used as stated in TODO, as in future user will be able to manage multiple applications